Dockerfile.dev

```diff
@@ -31,24 +23,23 @@ RUN \
         libxml2 \
         git \
         cmake \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+        # Needed by the stream component \
+        ffmpeg; \
+    rm -rf /var/lib/apt/lists/*
 
-WORKDIR /usr/src
+# Disable pip version check as we pin its version in requirements.txt
+ENV PIP_DISABLE_PIP_VERSION_CHECK=1
 
 # Setup hass-release
-RUN git clone --depth 1 https://github.com/home-assistant/hass-release \
-    && pip3 install -e hass-release/
-
-WORKDIR /workspaces
+WORKDIR /usr/src/hass-release
+RUN git clone --depth 1 https://github.com/home-assistant/hass-release . \
+    && pip3 install -e .
 
 # Install Python dependencies from requirements
-COPY requirements.txt ./
+WORKDIR /usr/src/homeassistant
+COPY requirements.txt .
 COPY homeassistant/package_constraints.txt homeassistant/package_constraints.txt
 RUN pip3 install -r requirements.txt --use-deprecated=legacy-resolver
```

Comment:
I don't think running this is needed anymore, since we take care of that via our setup/bootstrap scripts.

Reply:
setup/bootstrap isn't cached as part of docker build. I think it's helpful to keep it.

Reply:
> setup/bootstrap isn't cached as part of docker build. I think it's helpful to keep it.

It is also one of the most useless files to cache. They change very often. 
The devcontainer configuration provides means to differentiate between updating and rebuilding a container. 

Futhermore, for codespaces, we pre-built the full result (including the scripts defined in the devcontainer configuration).

Reply:
Got it. Ok.